lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -259,73 +259,115 @@ def _get_converged_core_config(service_mconfig: object) -> bool:
 
 def _get_default_slice_service_type_config(service_mconfig: object) -> str:
     """
-    Retrieves default_slice_service_type config value,If it does not exist
+    Retrieve default_slice_service_type config value,If it does not exist
     it defaults to 1.
     Args:
         service_mconfig:
 
     Returns: slice service type value
-    """
+    """  # noqa: DAR101, DAR201
```

Comment:
what is this `# noqa: DAR101, DAR201`?

Reply:
This stops, the doc string error which was coming for no reason.